README.md

```diff
@@ -513,67 +513,116 @@ from events
 group by 1
 ```
 
-## Iceberg
+#### Iceberg
 
-The adaptor support Iceberg as an Open Table Format. In order to work with Iceberg,
-setup an Iceberg connector and then create an Iceberg connection in Glue connections.
-Then use the following `conf` in your profile
+**Usage notes:** The `merge` with Iceberg incremental strategy requires:
+- To attach the AmazonEC2ContainerRegistryReadOnly Manged policy to your execution role :
+- To add the following policy to your execution role to enable commit locking in a dynamodb table (more info [here](https://iceberg.apache.org/docs/latest/aws/#dynamodb-lock-manager))
 ```
-connections: iceberg_connection
-conf: "spark.sql.catalog.iceberg_catalog=org.apache.iceberg.spark.SparkCatalog --conf spark.sql.catalog.iceberg_catalog.warehouse=s3://your_bucket/prefix --conf spark.sql.catalog.iceberg_catalog.catalog-impl=org.apache.iceberg.aws.glue.GlueCatalog --conf spark.sql.catalog.iceberg_catalog.io-impl=org.apache.iceberg.aws.s3.S3FileIO --conf spark.sql.extensions=org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions --conf spark.sql.sources.partitionOverwriteMode=dynamic --conf spark.sql.iceberg.handle-timestamp-without-timezone=true"
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "CommitLockingTableFullAccess",
+            "Effect": "Allow",
+            "Action": "dynamodb:*",
```

Comment:
Could you please add only the required policies rather than * ?

Reply:
Sure, I'll do it in the next PR.